android/app/src/main/java/app/organicmaps/location/LocationHelper.java

```diff
@@ -32,11 +36,14 @@
 import java.util.LinkedHashSet;
 import java.util.Set;
 
-public class LocationHelper implements BaseLocationProvider.Listener
+public class LocationHelper
+    implements BaseLocationProvider.Listener,
+               AppStateListener
 {
   private static final long INTERVAL_FOLLOW_MS = 0;
   private static final long INTERVAL_NOT_FOLLOW_MS = 3000;
   private static final long INTERVAL_NAVIGATION_MS = 0;
+  private static final long INTERVAL_TRACK_RECORDING_BACKGROUND = 10000;
```

Comment:
I suggest not wasting time on it now and always use the best precision/interval (zero). It will provide a good understanding and base raw data/stats that we can rely on later if decide to improve something.

Note that any OSMer or driver/cycler expects the track to be recorded with the max possible precision.